bucket/trid.json

```diff
@@ -0,0 +1,24 @@
+{
+    "homepage": "http://mark0.net/soft-trid-e.html",
+    "version": "v2.24-19.07.18",
```

Comment:
Shouldn’t `dd-mm-yy` be `yy-mm-dd` for semantic reasons?Should the dash be a period?

Reply:
And `v` should be removed 👍